network-map-viewer/src/components/network/network-map.js

```diff
@@ -457,16 +426,14 @@ const NetworkMap = (props) => {
     };
 
     const renderOverlay = () => (
```

Comment:
It's not a good practice to use variable to store component React doesn't tag this as react component and re-render it more than needed.